Automattic/wp-calypso
# Update design-preview UI + bump @wordpress/edit-site to 4.15.0

#### Proposed Changes

This diff updates the styles section of the design preview.
| Before | After |
| --- | --- |
|  ![Screen Shot 2022-10-19 at 5 38 28 PM](https://user-images.githubusercontent.com/797888/196655761-83d38bd7-bb7d-4f9d-ac91-1d7ed898f967.png)  |  ![Screen Shot 2022-10-19 at 5 39 48 PM](https://user-images.githubusercontent.com/797888/196655967-afd4e562-0d0f-47ad-a77c-e066daddbf4c.png) |


This diff also updates `@automattic/design-preview` dependency of `@wordpress/edit-site` from `4.6.0` to `4.15.0`, which contains the fixes:
- https://github.com/WordPress/gutenberg/pull/43601
- https://github.com/WordPress/gutenberg/pull/44556

Which, in turn, fixes this UI issue for the `<DesignPreview />` component:
| Before | After |
| --- | --- |
| ![Screen Shot 2022-10-19 at 5 44 11 PM](https://user-images.githubusercontent.com/797888/196657111-41f92fe4-9053-4faf-957b-d9634ce1380c.png) | ![Screen Shot 2022-10-19 at 5 44 38 PM](https://user-images.githubusercontent.com/797888/196657125-ddca33d8-1989-4779-894a-529b57d1eb2f.png) |


#### Testing Instructions



* Head to the design picker `setup/designSetup?siteSlug=${site_slug}`
* Preview any design with style variations.
* Ensure that the copy is updated as described above.
* Ensure that the style preview doesn't have extra padding (happened previously for the theme "Pixl"). 

#### Pre-merge Checklist
